src/routes/contactus/+page.svelte

```diff
@@ -1,110 +1,64 @@
 <script>
-	import { A, Avatar, Heading, P } from 'flowbite-svelte';
 	import { base } from '$app/paths';
-
-	const avatarStyle = 'w-20 rounded-full';
+	import Card from '../../lib/components/Card.svelte';
 </script>
 
-
-<div class="bg-white py-8">
-	<div class="mx-auto max-w-7xl gap-x-8 gap-y-20 px-6 lg:px-8">
+<div class="bg-white">
```

Comment:
I'd say get rid of this outside div because the padding in the layout adds unnecessary spacing on this page and looks weird. Later in the future, we'll redesign the layout file to make things look sharper and cleaner but for now, I'd advise getting rid of the white div.